### PR TITLE
my_hw_02

### PR DESCRIPTION
**Pull request by Shkuryn**
created a new branch on mentor's advice

### CODE REVIEW

**Review by B0RlS — COMMENTED** [review]
Could you please rename PR and make informative description please?

File: HW02/ihar_shkuryn/hw02.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MyArrayMethods
+  refine Array do
+    def my_map
+      if block_given?
+        results = []
+        i = 0
```
Comment (B0RlS) [inline HW02/ihar_shkuryn/hw02.rb:8]: try to avoid naming like `I`, `x` and etc.
You should use more informative variables naming

Comment (Shkuryn) [reply HW02/ihar_shkuryn/hw02.rb]: done

**Comment by Shkuryn** [comment]
Boris gave me permission

**Comment by Shkuryn** [comment]
Boris gave me permission

**Pull request closed without merging**